package.json

```diff
@@ -1531,6 +1531,15 @@
 							"altText": "create component from workspace folder"
 						}
 					},
+					{
```

Comment:
For here, it would be nice to either do one item that explains all 3 workflows or create a separate item for each workflow.